modules/javafx.controls/src/main/java/javafx/scene/control/ListCell.java

```diff
@@ -338,6 +338,7 @@ public void changed(
             updateSelection();
             updateFocus();
         }
+        updateEditing();
     }
```

Comment:
forgot yesterday: to keep consistent with TreeTableCell (and TreeCell), the updateEditing should be moved into the else-block (as last call) - couldn't find any difference (in fact couldn't reproduce the error the if/else is supposed to solve) as long as updateEditing behaves, though.

Reply:
I've moved it to the if-else. Probably doesn't matter.